.github/workflows/fullnode-execute-devnet-main.yaml

```diff
@@ -37,6 +37,18 @@ jobs:
             /tmp/node_log
           retention-days: 14
 
+      - name: Post to a Slack channel on failure
+        if: failure()
+        id: slack
+        uses: slackapi/slack-github-action@936158bbe252e9a6062e793ea4609642c966e302 # pin@v1.21.0
+        with:
+          payload: |
+            {
+              "text": "${{ job.status == 'success' && ':white_check_mark:' || ':x:' }} `${{ github.job }}`: <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|link>"
```

Comment:
`job.status` is always `failed` at this point, so we could simplify?

Reply:
This is copy/pasta from the forge jobs. I'm going to leave it as-is and then once all the tests land, just send out a big PR to update them all 😄 There's also duplication that needs to be cleaned up, too.